### ISSUE TITLE
Cleanup packages

### ISSUE THREAD

**Issue by clearab** [microsoft/botframework-components#874]
Our HelpAndCancel and Welcome packages contain quite a few blank files and empty folders, which I believe are artifacts of creating the dialogs through Composer. Can we cleanup these extraneous artifacts and remove them from the packages?

**Comment by clearab** [comment]
Also, why the 'exported' folder? It too seems extraneous.

**Comment by clearab** [comment]
@ryanlengel can you take a look at this one as well?

**Comment by peterinnesmsft** [comment]
@clearab @ryanlengel on the 'exported' folder, this was originally a pattern that was set by the documentation on creating packages to be consumed through Composer/Package Manager. @benbrown can you confirm if the exported folder is actually needed or not? If not, then I agree it should be dropped.

**Comment by ryanlengel** [comment]
As for the blank files, you are correct that these are artifacts from creating through Composer. For R12 we only had the necessary files but when @gabog took a pass at them the decision was made to include them. I don't believe that Composer generated them when importing the files otherwise, which may have led to errors. Gabo can you confirm?

**Comment by gabog** [comment]
Correct @ryanlengel, I was getting exceptions if these files weren't there, based on conversations with @benbrown, composer needs these empty files.

**Comment by clearab** [comment]
How is that supposed to work then? Are there restrictions like "If you want to add a dialog, you must add all of these exact files (even if they are empty) or bad things"? Are those bad things caused by Composer, or by dialog:merge? What if I only want to include a single specific file, rather than a full set?

**Comment by benbrown** [comment]
Yes, the `exported/` folder is required if you want the dialogs to appear in Composer. Only files inside this folder are copied into the project.

Composer expects all applicable files to exist, even if blank.

**Comment by clearab** [comment]
What does "all applicable files" mean? What if I want to include only something like a qna file, or an lg file?

**Comment by clearab** [comment]
Also, why the exported folder? Why wouldn't it just look anywhere in the package?

**Comment by clearab** [comment]
@benbrown @peterinnesmsft @ryanlengel @gabog can we please all discuss this one at the standup tomorrow? I'd like to make sure we have a shared understanding of this.

**Comment by benbrown** [comment]
> Also, why the exported folder? Why wouldn't it just look anywhere in the package?

So that you can control what parts of the component a developer can modify.

**Comment by peterinnesmsft** [comment]
Confirmed with @ryanlengel that this has been postponed to R14 - moving to backlog.
